README.txt

```diff
@@ -1 +1 @@
-The ICAPS 2017 conference website, based on Halcyonic by HTML5 UP
+The ICAPS 2017 conference website, based on the ICAPS-16 website, which in turn was based on Halcyonic by HTML5 UP

```

Comment:
Just as a minor note, the html originally came from the Halcyonic HTML5 UP template, but was dramatically overhauled and reworked several times and in several ways. I had a large hand in content, but design was largely due to [Kristie Taylor-Muise](http://www.kristietaylor-muise.ca/project/icaps-conference-website/).

PS. Brilliant idea with the pull requests ;).